Source/items.cpp

```diff
@@ -2007,8 +2010,13 @@ int RndUItem(int m)
 			if (monster[m].mLevel < AllItemsList[i].iMinMLvl)
 				okflag = FALSE;
 		} else {
+#ifdef HELLFIRE
+			if (2 * curlv < AllItemsList[i].iMinMLvl)
+				okflag = FALSE;

```

Comment:
The other `ifdef` in this function only contain the the if-statment and not the body

Reply:
fixed